tests/e2e/performance/io_workload/test_fio_benchmark.py

```diff
@@ -176,7 +176,7 @@ def setting_storage_usage(self):
         # To make sure the number of App pods will not be more then 50, in case
```

Comment:
please change this number as well, since the new value will reduce the number of servers to 24 ? 25 ?